src/TensorOperationsTBLIS.jl

```diff
@@ -1,93 +1,34 @@
 module TensorOperationsTBLIS
 
 using TensorOperations
+using TensorOperations: StridedView, DefaultAllocator, IndexError
+using TensorOperations: istrivialpermutation, BlasFloat, linearize
+using TensorOperations: argcheck_tensoradd, dimcheck_tensoradd,
+                        argcheck_tensortrace, dimcheck_tensortrace,
+                        argcheck_tensorcontract, dimcheck_tensorcontract
 using TensorOperations: Index2Tuple, IndexTuple, linearize, IndexError
-using LinearAlgebra: BlasFloat, rmul!
+using LinearAlgebra: BlasFloat
 using TupleTools
 
-include("LibTblis.jl")
-using .LibTblis
+include("LibTBLIS.jl")
+using .LibTBLIS
+using .LibTBLIS: LibTBLIS, len_type, stride_type
 
-export tblis_set_num_threads, tblis_get_num_threads
-export tblisBackend
+export TBLIS
+export get_num_tblis_threads, set_num_tblis_threads
+
+get_num_tblis_threads() = convert(Int, LibTBLIS.tblis_get_num_threads())
+set_num_tblis_threads(n) = LibTBLIS.tblis_set_num_threads(convert(Cuint, n))
```

Comment:
Yes I also considered that and would be fine with it. It would have been a bit nicer if it was something like `TBLIS.set_num_threads`, but that is harder to accomplish. But I do like that I know it will be something with `set_num` and then I can let TAB autocomplete take over.